site/app/repositories/chat/ChatroomRepository.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace app\repositories\chat;
+
+use app\entities\chat\Chatroom;
+use Doctrine\ORM\EntityRepository;
+
+class ChatroomRepository extends EntityRepository {
+
+    public function findAllChatroomsByHostId(string $hostId) {
+        return $this->createQueryBuilder('c')
+                    ->where('c.host_id = :hostId')
+                    ->setParameter('hostId', $hostId)
+                    ->getQuery()
+                    ->getResult();
+    }
+
+    public function findAllActiveChatrooms() {
+        return $this->createQueryBuilder('c')
+                    ->where('c.isActive = :isActive')
+                    ->setParameter('isActive', true)
+                    ->getQuery()
+                    ->getResult();
+    }
+
+    public function findAllInactiveChatrooms() {
+        return $this->createQueryBuilder('c')
+                    ->where('c.isActive = :isActive')
+                    ->setParameter('isActive', false)
+                    ->getQuery()
+                    ->getResult();
+    }
```

Comment:
I believe it's recommended to use DQL instead of the query builder for performance.